package.json

```diff
@@ -21,6 +21,7 @@
     "lint:fix": "yarn lint:typescript --fix",
     "lint:typescript": "eslint --max-warnings=0 ./src -c ./.eslintrc.js",
     "manage-translations": "node ./translations/translation-runner.js",
+    "nigh-android": "react-native run-android --variant=devDebug --appIdSuffix=nightly",
```

Comment:
ios version doesn't work i need to review why

Reply:
`run-ios` uses `--scheme` not `--variant` maybe is this. But I'd recommend to add it in another PR since is not part of the fix. `react-native run-ios --scheme=emurgo-staging --configuration=Debug`